external/llvm-project/mlir/lib/Target/LLVMIR/Dialect/ROCDL/ROCDLToLLVMIRTranslation.cpp

```diff
@@ -97,8 +116,36 @@ class ROCDLDialectLLVMIRTranslationInterface
           moduleTranslation.lookupFunction(func.getName());
       llvm::SmallString<8> llvmAttrValue;
       llvm::raw_svector_ostream attrValueStream(llvmAttrValue);
-      attrValueStream << "1, " << value.getInt();
+      attrValueStream << "1," << value.getInt();
+      llvmFunc->addFnAttr("amdgpu-flat-work-group-size", llvmAttrValue);
+    }
+    // Set max-flat-work-group-size from known block sizes if set
```

Comment:
[Question] Would you mind to explain a bit more of where you find this attribute, where it is defined and why it is necessary to set it now?

Reply:
... that is a typo in the comment, though.

This attribute is something used an the AMDGPU backend. In at least one codepath, it's a fallbacak from when the `!reqd_work_group_size` isn't set, but I want to ensure we have consistency.